firefox-ios/Shared/Strings.swift

```diff
@@ -4916,27 +4916,27 @@ extension String {
             key: "Menu.OpenBookmarksAction.AccessibilityLabel.v2",
             tableName: "Menu",
             value: "Bookmarks",
-            comment: "Accessibility label for the button, displayed in the menu, used to open the Bookmarks home panel.")
+            comment: "Accessibility label for the button, displayed in the menu, used to open the Bookmarks home panel. Please keep as short as possible, <15 chars of space available.")
```

Comment:
Another divergence in these strings. Which one is correct?

It doesn't look right for accessibility strings to have a length limit.

Reply:
@dicarobinho @adudenamedruby 
Looks like this part was removed in https://github.com/mozilla-mobile/firefox-ios/pull/22331

So, updating with the assumption that the comment shouldn't have the last part.